### ISSUE TITLE
guzzlehttp/psr7 ^2

### ISSUE THREAD

**Issue by kw-pr** [thephpleague/omnipay-tests#21]
You require `guzzlehttp/psr7 ^1` but current version is 2.4.1.

Please consider upgrading or allow both versions:

    $ composer require omnipay/tests --dev
    ...
    Problem 1
    - Root composer.json requires omnipay/tests ^4.1 -> satisfiable by omnipay/tests[v4.1.0, v4.1.1, 4.1.x-dev].
        - omnipay/tests[v4.1.0, ..., 4.1.x-dev] require guzzlehttp/psr7 ^1 -> found guzzlehttp/psr7[1.0.0, ..., 1.x-dev] but the package is fixed to 2.4.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Thanks for all the good work!

If you like I could make a PR.

**Comment by DemigodCode** [comment]
Yeah, getting security alerts for my vr-payment package from dependabot.
Would be nice to update to the new version.
